packages/cli/cli/versions.yml

```diff
@@ -1,3 +1,11 @@
+- changelog_entry:
+  - summary: |
+      Fix an issue where some environment variables were not substituted before generating the IR.
```

Comment:
nit: the user facing change here is probably more like

Fix an issue where some postman environment variables (e.g. `api key`) were not substituted when running `fern generate`.

Reply:
Sweet will update!